public/static/js/version_admin.js

```diff
@@ -76,8 +76,10 @@ mciModule.controller('AdminOptionsCtrl', [
                     $scope.closeAdminModal()
                     $rootScope.$broadcast("version_updated", data)
                     notifier.pushNotification(
-                      "Version " + (active ? "scheduled." : "unscheduled.") +
-                      (abort ? "\n In progress tasks will be aborted." : ""),
+                      "Version " + (active ? "scheduled." : "unscheduled") +
```

Comment:
Do we need to address if an individual build or task is deactivated?

Reply:
I think we might not need to address tasks since deactivated tasks won't prevent the build from being complete.
https://github.com/evergreen-ci/evergreen/blob/612c4407f3065a13c95d1f1a1d79906782075564/model/build/build.go#L98

but deactivating builds could still cause it, I think.